src/main/main.ts

```diff
@@ -58,21 +58,20 @@ export { DeviceCredential, LoginWithWebAuthnParams, SignupWithWebAuthnParams } f
 export interface Config {
   clientId: string
   domain: string
+  /**
+   * Used to define which Google provider variant to use with Google One Tap
+   * @default "default"
+   * */
+  googleVariant?: string
   language?: string
   locale?: string
   webAuthnOrigin?: string
 }
 
-export type ApiClientConfig = {
+export type ApiClientConfig = RemoteSettings & {
   clientId: string
-  language?: string
-  scope?: string
-  orchestrationToken?: OrchestrationToken
-  sso: boolean
-  pkceEnforced: boolean
-  isPublic: boolean
   baseUrl: string
-  googleClientId?: string
+  orchestrationToken?: OrchestrationToken
 }
```

Comment:
`ApiClientConfig` is built from response of remote staggings API response.
See line 163-168 of this file.